### PR TITLE
Remove EscalationPolicy model registration

### PR DESCRIPTION
**Pull request by jarrydlee**
I removed the model registration here and made it so that you can import the schema using 
```import EscalationPolicySchema from '../../models/escalationPolicy';```.

@kalebdavis

### CODE REVIEW

**Comment by jarrydlee** [comment]
Sad path added to test cases @kalebdavis

File: server/tests/models/escalationPolicy.test.js [hunk 1]
```diff
@@ -1,20 +1,25 @@
-import { expect } from 'chai';
-import EscalationPolicy from '../../models/EscalationPolicy';
+import chai, { expect } from 'chai';
+import mongoose from 'mongoose';
+import EscalationPolicySchema from '../../models/escalationPolicy';
+
+chai.config.includeStack = true;
```
Comment (kalebdavis) [inline server/tests/models/escalationPolicy.test.js]: Nice, is there a way we can make this the default?

**Comment by kalebdavis** [comment]
👍 lgtm

File: server/tests/models/escalationPolicy.test.js (hunk 1 above)
Comment (jarrydlee) [inline server/tests/models/escalationPolicy.test.js]: I'll have a look, I'll create another PR if we can

**Pull request merged**